docs/src/prototypes/Prototypes.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import { Box, Header, Segment } from '@stardust-ui/react'
+
+interface PrototypeSectionProps {
+  title?: string
```

Comment:
```suggestion
  title?: React.ReactNode
```

Please allow to pass elements.

Reply:
done, thx